spec/policies/user_policy_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserPolicy do
+  subject { described_class.new(current_user, user) }
+
+  let(:current_user) { nil }
+  let(:user) { create(:user) }
+
+  context 'with a current_user' do
```

Comment:
Here (and, similar in some other policy specs): Wouldn't we need to test the policy without a user as well (i.e., ensuring that it doesn't allow anything)?

Reply:
I thought we can omit those because the application_policy_spec.rb already tests that and for the policies that override the standard behavior we have these tests. However since it does not really cost anything and its just a couple lines I added a test for all policies.

Reply:
Perfect, thank you! In general, I would agree with your optimisation and agree that testing the super class / method would be enough. However, for this security context, I'd rather opt to duplicate a few lines in tests to make it very explicit, since this will raise awareness if anything changes unexpectedly.